core/iwasm/libraries/libc-uvwasi/FindLIBUV.cmake

```diff
@@ -0,0 +1,25 @@
+# Find libuv library
```

Comment:
Should add license header, please refer to other cmake file

Reply:
Done